tests/integration/apps/test_flush.py

```diff
@@ -0,0 +1,78 @@
+import unittest
+from parameterized import parameterized
+import lusid
+import pathlib
+import tests.unit.apps.test_data.test_transactions as txn_test_data
+import lusidtools.apps.flush_transactions as flush
+import datetime
+from dateutil.tz import tzutc
+import logging
+
+
+class FlushAndFillIntegrationTests(unittest.TestCase):
```

Comment:
Is it possible to move this into the [AppTests class](https://github.com/finbourne/lusid-python-tools/blob/7da8bd54b93c4b41b8c3fd1caeac17bf036e2538/tests/integration/apps/test_command_line_apps.py#L20) which is specifically for the CLI apps? You'll benefit from using some of the test fixtures set out there.